recipes/doctest/2.3.5/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class DoctestConan(ConanFile):
+    name = "doctest"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/onqtam/doctest"
+    description = "C++11/14/17/20 single header testing framework"
+    license = "MIT"
+    settings = ("os", "compiler", "build_type", "arch")
+    generators = "cmake"
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    @property
+    def _is_mingw(self):
+        return self.settings.os == "Windows" and self.settings.compiler == "gcc"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["DOCTEST_WITH_TESTS"] = False
+        cmake.configure(build_folder=self._build_subfolder, source_folder=self._source_subfolder)
+        return cmake
+
+    def build(self):
```

Comment:
Is this step needed as this is a header-only library?

Reply:
I used to think it was, since the single header generation is a target that must be built.

But since we're retrieving the release artifacts, the single header is already present. will remove